.github/workflows/containerize-invehicle-stack-agemo.yml

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: MIT
+
+name: Containerize In-Vehicle Stack - Agemo
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+      - name: Set up Docker buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          install: true
+
+      - name: Login to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Checkout Agemo
+        uses: actions/checkout@v2
+        with:
+          repository: eclipse-chariott/Agemo
+          submodules: recursive
+          ref: 0.1.2
+
+      - name: Build and push multi-platform Docker image for Mosquitto image with Agemo config
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: Dockerfile.mosquitto.multi
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ghcr.io/eclipse-sdv-blueprints/software-orchestration/eclipse-mosquitto/mqtt-broker-agemo-config:0.1.2

```

Comment:
I am thinking about, shall we define the version tag one times e.g. as Github action env variable to ensure to reduce possibility of making mistakes. Someone might forget to adjust the image version tag when checking out another agemo version. To define this once would keep it automatically in sync. We could do this for all image builds as well, if you like the idea.

Reply:
I have seen that you have accepted it and pushed it, but unfortunately I am not able to close the conversation.